### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#197] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-20 224033](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/f9ec93a5-ea9a-433c-b546-824719465a5f)

**Comment by IonutCicio** [comment, marked as answer]
## Soluzione confermata in classe
Siano $X \sim \text{Bern}(p), Y \sim \text{Bern}(p)$, e $Z = X(1 - Y), W = 1 - XY$

Consideriamo i vari casi per $Z$ e $W$

|  z  | 0 | 1 |
|--|--|--|
| **0** | 0 | 1 |
| **1** | 0 | 0 |

| w  | 0 | 1 |
|--|--|--|
| **0** | 1 | 1 |
| **1** | 1 | 0 |

Ora si può vedere la distribuzione congiunta di $\mathbb{P}(Z=z, W=w)$, per cui bisogna vedere i 4 casi. Ad esempio $W = 0 \land Z = 0 \iff X = 1 \land Y = 1 \implies \mathbb{P}(W = 0, Z = 0) = p^2$

| $\frac{W}{Z}$  | 0 | 1 |
|--|--|--|
| **0** | $p^2$ | $p(1-p) + (1-p)^2$ |
| **1** | $0$ | $p(1-p)$ |

Ora ricavare la distribuzione marginale è facile, basta fare la somma delle righe e delle colonne

| $\frac{W}{Z}$  | 0 | 1 | $W$ |
|--|--|--|--|
| **0** | $p^2$ | $p(1-p) +(1-p)^2$ | $p^2 + p(1-p) + (1-p)^2$ |
| **1** | $0$ | $p(1-p)$ | $p(1-p)$ |
| $Z$ |  $p^2$ |  $(1-p)^2 + 2p(1-p)$ | |

Per quanto riguarda i valori marginali, la tabella così è un po' difficile da visualizzare, quindi riscrivo le distribuzioni marginali anche per casi, ma è la stessa cosa

$$
\mathbb{P}(Z=z) =
\begin{cases}
p^2 + p(1-p) + (1-p)^2 & \text{se } z = 1\\\\
p(1-p) & \text{se } z = 0
\end{cases} =
\begin{cases}
p^2 - p + 1 & \text{se } z = 1\\\\
p(1-p) & \text{se } z = 0
\end{cases}
$$

$$
\mathbb{P}(W=w) =
\begin{cases}
(1-p)^2 + 2p(1-p) & \text{se } w = 1\\\\
p^2 & \text{se } w = 0
\end{cases} =
\begin{cases}
1-p^2& \text{se } w = 1\\\\
p^2 & \text{se } w = 0
\end{cases}
$$


Sono indipendenti per $p = 0, 1$, perché in quel caso sono **variabili aleatorie certe**, e sono **sempre indipendenti**. Si dimostra che $W, Z$ non sono indipendenti per nessun altro valore. Basta considerare il caso $Z = 1, W = 0$, il prodotto deve fare $0$, e gli unici valore per cui $p(1-p)p^2 = (p - p^2)p^2 = p^3 - p^4 = p^3(1 - p) = 0$ sono $1$ e $0$
